kimchi/src/circuits/expr.rs

```diff
@@ -1984,11 +1984,11 @@ where
         let mut res = String::new();
         for (k, v) in env.into_iter() {
             let rhs = v.ocaml_str();
-            let cached = format!("let {} = {rhs} in", k.var_name());
+            let cached = format!("let {} = {rhs} in ", k.var_name());
```

Comment:
I think this little change will actually annoy the formatter on the ocaml side :o